src/hydrusvideodeduplicator/dedup.py

```diff
@@ -237,6 +238,29 @@ def clear_search_cache() -> None:
         except OSError:
             rprint(f"[red] Database does not exist. Cannot clear search cache.")
 
+    # Updates the 'farthest_search_index' of all videos in the database that have an index greater than the length of
+    # the database, reducing them down to the length of the database.
+    @staticmethod
+    def update_search_cache(new_total: int | None = None) -> None:
+
+        if new_total is not None and new_total < 1:
```

Comment:
This should be an assert. This can only happen from a programming error, not from outside influence from the db or anything else. The exception message is pretty useless because it just restates the logic with words